### PR TITLE
Dma3 bugfixing

### PR DESCRIPTION
**Pull request by Dat-NguyenDuy**
**Prerequisites**

- [ x] I have checked latest main branch and the issue still exists.
- [ x] I did not see it is stated as known-issue in release notes.
- [ x] No similar GitHub issue is related to this change.
- [x ] My code follows the commit guidelines of this project.
- [ x] I have performed a self-review of my own code.
- [ x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.

**Describe the pull request**

**Type of change**

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

**Tests**

- Test configuration (please complete the following information):
  - Hardware setting:
  - Toolchain:
  - Test Tool preparation:
  - Any other dependencies:
- Test executed
  Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce.
  - [ ] Build Test
  - [ ] Run Test

### CODE REVIEW

**Comment by mmahadevan108** [comment]
@mcuxsusan, could you kindly review.

File: drivers/dma3/fsl_edma.c [hunk 1]
```diff
@@ -924,9 +924,9 @@ void EDMA_PrepareTransferConfig(edma_transfer_config_t *config,
     assert(srcAddr != NULL);
     assert(destAddr != NULL);
     assert((srcWidth == 1U) || (srcWidth == 2U) || (srcWidth == 4U) || (srcWidth == 8U) || (srcWidth == 16U) ||
-           (srcWidth == 32U));
+           (srcWidth == 32U) || destWidth == 64U);
```
[the remark this answers is not in the document]
Comment (Dat-NguyenDuy) [reply drivers/dma3/fsl_edma.c:927]: yeah, right should be `srcWidth`. I'm building the mcux hal with assertion enabled, then without this there would be an assertion when i tried to configure 64bytes data size for each DMA transfer

[the remark this answers is not in the document]
Comment (Dat-NguyenDuy) [reply drivers/dma3/fsl_edma.c]: I've updated

[the remark this answers is not in the document]
Comment (mcuxted) [reply drivers/dma3/fsl_edma.c]: Okay, I have no questions about this patch.

[the remark this answers is not in the document]
Comment (mcuxsusan) [reply drivers/dma3/fsl_edma.c]: > Okay, I have no questions about this patch.

Hi @mcuxted, if you are OK with the change, please approve the review then I will help merge it.

**Review by mcuxted — APPROVED** [review]

**Pull request merged**